mock-core-configs/mock-core-configs.spec

```diff
@@ -89,7 +89,10 @@ cp -a etc/mock/eol/templates/*.tpl %{buildroot}%{_sysconfdir}/mock/eol/templates
 # generate files section with config - there is many of them
 echo "%defattr(0644, root, mock)" > %{name}.cfgs
 find %{buildroot}%{_sysconfdir}/mock -name "*.cfg" -o -name '*.tpl' \
+    | grep -v chroot-aliases \
```

Comment:
Never rely on unqualified "grep" statements, they can accidentally filter peculiar names like "epel-8-test-chroot-aliases.cfg", and hilarity could ensue. And *always* put a dangling slash on a directory name, in case someone used a symlink for /etc/mock for whatever reason.

      ind %{buildroot}%{_sysconfdir}/mock/ -name 'chroot-aliases -prune -o -type f -name "*.cfg" -print -o -print -name '*.tpl' -print \

Reply:
Oh, have you been bitten by this?  Note that we talk about `%{buildroot}` which is automatically cleaned by rpmbuild/mock when building Mock rpm. Such an unexpected file shouldn't exist at all.

Even if, this shouldn't cause any big headaches to mock package maintainers because the build log clearly says what is happening.  If you believe this is worth fixing, open a PR.